### PR TITLE
Added a script to validate custom constitution

### PR DESCRIPTION
**Pull request by msftsettiy**
A standalone script to validate operator actions on a CCF network. The script will run a few tests to validate if an operator member (a member with the member data attribute 'is_operator' set to true) could perform certain governance actions.

### CODE REVIEW

**Comment by andpiccione** [comment]
The script would be duplicated with the one we already have here: https://github.com/microsoft/mccf-vscode-extension/blob/main/src/scripts/test_operator_actions.sh. 

I think we need to find a common place to keep it, without duplicating. If you prefer to keep it under `src/tools`, we can remove the one under the `scripts` dir and then copy it during the creation of the CCF project. But it would require some changes in the extension code and/or configs.

File: src/tools/test_operator_actions.sh [hunk 1]
```diff
@@ -0,0 +1,330 @@
+#!/bin/bash
+trap cleanup EXIT
+
+set -euo pipefail
+
+declare certificate_dir=$(mktemp -d)
+declare signing_cert=""
+declare signing_key=""
+declare server=""
+declare proposals_dir=$(mktemp -d)
```
Comment (weirenlai) [inline src/tools/test_operator_actions.sh]: I think as you change the `proposals_dir` here, you need to copy the json files to this directory before testing.

Comment (msftsettiy) [reply src/tools/test_operator_actions.sh]: The proposals are self-contained in the script.

Comment (weirenlai) [reply src/tools/test_operator_actions.sh]: Oh, I didn't notice that, thanks for clarifying.

**Review by weirenlai — DISMISSED** [review]

**Review by weirenlai — APPROVED** [review]

**Pull request merged**